decide/booth/static/js/grafica_visualizer.js

```diff
@@ -0,0 +1,124 @@
+var ctx1 = document.getElementById('bar-resultados').getContext('2d');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
var ctx1 = document.getElementById("bar-resultados").getContext('2d');
```